setup.py

```diff
@@ -46,8 +46,9 @@
         'Flask==0.12.2',
         'Werkzeug==0.12.2',
     ],
-    'build': [
-        'docker==2.5.1'
+    'cli': [
+        'docker==2.5.1',
+        'PyYAML>=3.12',
```

Comment:
Could we agree on not using `PyYAML`, and standardize on `ruamel.yaml` instead? My understanding is that the latter is the preferred way to interact with YAML in Python nowadays.

Reply:
I generally prefer `ruamel.yaml` as well; however, in this case I picked `PyYAML` specifically because it is required by [awscli](https://github.com/aws/aws-cli/blob/develop/setup.py#L31) which might be used with this tool. I thought us requiring `ruamel.yaml` and `awscli` requiring `PyYAML` made for a silly situation where we were forcing someone to install 2 different YAML parsers for some. Considering that our YAML usage here is pretty basic, would you be agreeable to using `PyYAML` here for that reason?

Reply:
I'm going to go ahead and merged this since it is approved - however, if you would like to switch to `ruamel.yaml` please feel free to do so.

Reply:
The cli parser will try to look for an extra that has the same name as the command (i.e. build or run) and then ensure all the dependencies from that extra are installed. I'm okay switching to a single extra called `cli` that covers all commands, but if we go this way, then the logic that checks for the individual extras probably needs to go (see https://github.com/racker/fleece/blob/master/fleece/cli/main.py#L11-L19)

Reply:
@ryandub That sounds totally reasonable, thanks for the explanation. I didn't want to block this PR just because of this.

Reply:
Thanks @szilveszter . @miguelgrinberg - I forgot about that! Thanks for the catch. I will fix it before merging.